chart/scripts/post-delete-hook.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+echo "Preparing for uninstalling apps left by Rancher in the following namespaces: " $namespaces
+for namespace in $namespaces; do
+  # delete the helm operator pods
+  pod_list=$(kubectl get pods -n "$namespace" -o name --ignore-not-found)
+  for pod in $pod_list; do
+    if [[ $pod =~ ^pod\/helm-operation-* ]]; then
+      echo "--- deleting the pod [$pod]"
+      kubectl delete $pod -n "$namespace"
+    fi
+  done
+  app_list=$(helm list -n "$namespace" -q)
+  for app in $app_list; do
+    if [[ $app =~ .crd$ ]]; then
+      echo "--- skip" $app
+      continue
+    fi
+    echo "--- deleting the app [$app] (namespace: $namespace)"
+    helm uninstall "$app" -n "$namespace"
+    c=0
+    # wait 120 sections in total for an uninstallation to be finished
+    while [[ "$(kubectl get pods --selector app="$app" -n "$namespace" -o json | jq '.items | length')" != "0" ]] && [[ $c -lt 20 ]]; do
```

Comment:
Have you considered to set retries configurable from values?? I mean, here is hardcoded 20 retries every 6 seconds, 120 seconds. Would make sense to add a a new values var `.Values.postDelete.retries = 20` to make it configurable??

```
-    while [[ "$(kubectl get pods --selector app="$app" -n "$namespace" -o json | jq '.items | length')" != "0" ]] && [[ $c -lt 20 ]]; do
+    while [[ "$(kubectl get pods --selector app="$app" -n "$namespace" -o json | jq '.items | length')" != "0" ]] && [[ $c -lt $retries ]]; do
```